README.md

```diff
@@ -210,8 +210,7 @@ import Hello from '@/components/hello.vue'
 }
 ```
 
-6. The commonly used `@` path wildcard will work if you set up a `baseUrl` and `paths` (in `compilerOptions`) to include `@/*`. If you don't set this, then 
-the fallback for the `@` wildcard will be `[tsconfig directory]/src` (we hope to make this more flexible on future releases):  
+6. It accepts any wildcard in your TypeScript configuration:  
```

Comment:
Since we have this, the `wildcard` option in fork-ts-checker-webpack-plugin` was dropped right?

Reply:
Yeah, it looks like you dropped that option (good!) since I don't see it here in the final file changes. :stuck_out_tongue:

Reply:
My fault I was using Github "online" haha and every change was a commit